main.c

```diff
@@ -261,14 +262,13 @@ motion_notify(XEvent *e)
 }
 
 int
-main(int argc, char *argv[])
```

Comment:
At some point ill add these back for maybe some cli options but for now removing them gets rid of warnings so this is good